src/Search.js

```diff
@@ -0,0 +1,83 @@
+import React, { useState, useEffect } from 'react';
+import styled from 'styled-components';
+import { useParams, useHistory } from 'react-router-dom';
+
+const Section = styled.section`
+  text-align: center;
+  margin-top: 51px;
+`;
+
+const Title = styled.h1`
+  font-family: 'Bitter', serif;
+  font-weight: normal;
+  font-size: 38px;
+  letter-spacing: 0.8px;
+`;
+
+const Form = styled.form`
+  margin-top: 30px;
+`;
+
+const Span = styled.span`
+  font-size: 18px;
+  color: #9e9e9e;
+`;
+
+const Input = styled.input`
+  width: 350px;
+  height: 36px;
+  margin: 0 10px;
+  padding-left: 15px;
+  background: #ffffff;
+  border: 1px solid #e6e6e6;
+  border-radius: 2px;
+  font-size: 14px;
+  letter-spacing: 0.8px;
+`;
+
+const Button = styled.button`
```

Comment:
This component could be shared with the hero section component to avoid code duplication